ui/client/components/graph/nodeDetailsModal.js

```diff
@@ -132,7 +132,8 @@ class NodeDetailsModal extends React.Component {
             </div>
             {this.state.editedNode.nodes.map((node, idx) => (<div key={idx}>
                                     <NodeDetailsContent isEditMode={false} node={node} processDefinitionData={this.props.processDefinitionData}
-                                         nodeErrors={this.props.nodeErrors} onChange={() => {}} testResults={testResults(node.id)}/><hr/></div>))}
+                                         nodeErrors={this.props.nodeErrors} onChange={() => {}} testResults={testResults(node.id)}
+                                         processToDisplay={this.props.processToDisplay}/><hr/></div>))}
```

Comment:
I don't understand what u mean by this. How to "take if from reducer"? You mean to pass `graphReducer` instead of `processToDisplay` in props? Why it would be better? (Forgive me but I'm still not have the best intuition how to use redux).

Reply:
In nodeDetailsModal.js we inject props from reducer through mapState method. We can do the same in nodeDetailsContent.js

Reply:
I'm one step closer to understand redux - thank you! :-) I've also taken processDefinitionData from redux state.